SignalMessaging/utils/DisplayableText.swift

```diff
@@ -217,20 +217,20 @@ extension String {
     @objc
     public class func displayableText(_ rawText: String) -> DisplayableText {
         // Only show up to N characters of text.
-        let kMaxTextDisplayLength = 1024
+        let kMaxTextDisplayLength = 512
```

Comment:
I reduced the max length of text we should in conversation view, since we were previously showing more than a screenful of Latin text on even an iPhone X, which has a particularly long screen.  Check out the screenshots.

I've discussed this briefly with Myles, although there's no clear decision on the exact right length.